apisix/plugins/sls-logger.lua

```diff
@@ -63,7 +63,11 @@ local schema = {
 
 local metadata_schema = {
     type = "object",
-    properties = {log_format = log_util.metadata_schema_log_format,},
+    properties = {
+        log_format = {
+            type = "object"
+        }
+    },
```

Comment:
What does this change have to do with etcd?

Reply:
Originally sls-logger didn't add the metadata_schema, and then because the etcd checker didn't take effect before, it couldn't detect the problem.
As for the metadata schema, I refer to file-logger.
https://github.com/apache/apisix/blob/2f6fd50121c545742a902cf838287406d103566a/apisix/plugins/file-logger.lua#L63-L70